coding-the-legislation/10_basic_example.md

```diff
@@ -17,19 +17,30 @@ class flat_tax_on_salary(Variable):
 ```
 
 Let's explain in details the different parts of the code:
-- `class flat_tax_on_salary(Variable):` declares a new variable with the name `flat_tax_on_salary`.
-- Metadatas:
-  - `value_type = float` declares the type of the variable. Possible types are the basic python types:
+
+### The name of the variable
+
+`class flat_tax_on_salary(Variable):` declares a new variable with the name `flat_tax_on_salary` in accordance with our [naming conventions](../contribute/variables-naming.md).
+
+### The metadata
```

Comment:
Maybe call that `attributes` instead of `metadata` ?

The `metadata` we introduced in parameters are free fields that are not used by the code, which is not really the same that what is described here.